docs/api/app.md

```diff
@@ -530,12 +530,6 @@ Returns `String` - The current application locale. Possible return values are do
 
 **Note:** On Windows you have to call it after the `ready` events gets emitted.
 
-### `app.setLocale(locale)`
```

Comment:
What became of this method?

Reply:
nvm, I see: https://github.com/electron/electron/pull/11469#discussion_r166983107